### PR TITLE
SUBMARINE-546. Update Submarine Website for 0.4.0 release

### PR DESCRIPTION
**Pull request by jiwq**
New Website

### CODE REVIEW

File: _includes/themes/submarine/_navigation.html [hunk 1]
```diff
@@ -24,34 +25,27 @@
...
-          <ul class="dropdown-menu">
-            <li class="title"><span><b>Release</b><span></li>
-            <li><a href="https://hadoop.apache.org/submarine/docs/0.2.0/Index/">0.2.0</a></li>
-            <li><a href="http://hadoop.apache.org/submarine/docs/">0.1.0</a></li>
-            <li role="separator" class="divider"></li>
-              <li class="title"><span><b>Snapshot&nbsp;(development)</b><span></li>
-            <li><a href="/docs/0.3.0-SNAPSHOT">0.3.0-SNAPSHOT</a></li>
-          </ul>
-        </li-->
-
```
Comment (wangdatan) [inline _includes/themes/submarine/_navigation.html:43]: So we removed 0.3.0 download link?

Comment (jiwq) [reply _includes/themes/submarine/_navigation.html:43]: This snippet was commented out and it's not the download link, so we can delete it.

**Review by pingsutw — COMMENTED** [review]
+1, Thanks @jiwq 
I found that website doesn't update after #15 is merged.
We need to copy `_site` directory to `asf-site` branch
FRI: https://github.com/apache/submarine-site#deploy-website

**Comment by jiwq** [comment]
> I found that website doesn't update after #15 is merged.

Yes, we will publish the site when this PR is merged.

**Pull request merged**